doc/requirements.txt

```diff
@@ -32,4 +32,4 @@ torchvision==0.12.0; sys_platform == "darwin" and python_version == "3.10"
 jinja2==3.0.3
 retworkx==0.11.0
 networkx==2.6
-xanadu-sphinx-theme==0.1.0
+pennylane-sphinx-theme
```

Comment:
Can we pin here?
```suggestion
pennylane-sphinx-theme==0.3.1
```

Reply:
I deliberately do not pin, as this allows us to update the navbar/footer in a single repo, and have it automatically propagate to all other PL repos 🙂 

It's not ideal, but if we pin in each repo, we are back in a situation where we need to still update 16 different repos just to change the navbar!

Reply:
```suggestion
# we do not pin the sphinx theme, to allow the
# navbar and header data to be updated at the source
pennylane-sphinx-theme
```

Reply:
I think that sounds very reasonable! Could we have a comment here in the requirements file about that?

We've started following the strict pinning approach in the requirements files, so good to document the reason for having no pin here.